### PR TITLE
zh_CN.lang 

### PR DESCRIPTION
**Pull request by DYColdWind**

### CODE REVIEW

File: src/main/resources/assets/yoyos/lang/zh_CN.lang [hunk 1]
```diff
@@ -64,6 +64,12 @@ modifier.floating.desc=§o仲夏夜之梦！§r\n悠悠球将便得如气球一
 modifier.gardening.name=裁剪
 modifier.gardening.desc=§o精湛的牧羊人...§r\n收割所到之处的草与花，它甚至可以进行修剪羊毛。
 
+modifier.sticky.name=粘稠
```
Comment (Snownee) [inline src/main/resources/assets/yoyos/lang/zh_CN.lang]: Will "粘性" be better?

**Comment by Snownee** [comment]
Um.. Still have some questions:

* Translations seems to be wrong at line 6 & 7;
* Untranslated statements between line 11 and 14.

**Comment by DYColdWind** [comment]
@Snownee 
'Translations seems to be wrong at line 6 & 7;'
I know about this, but you do not feel translated‘裁剪悠悠球/粘液悠悠球’，is it hard to pronounce（有些拗口）?
‘Untranslated statements between line 11 and 14.’
I generally do not go to translate cfg
Moreover, they are only 'title'（标题）

**Comment by Snownee** [comment]
At least its not so hard to pronounce for me. Now the translations tend to be a attribute of Diamond & Iron Yoyo, but the fact is they are obviously different.

**Comment by Jozufozu** [comment]
If these look good to the two of you, then I'll pull. Thank you so much for the contribution!

**Comment by DYColdWind** [comment]
@Snownee

**Comment by Snownee** [comment]
@DYColdWind Nice work! Thanks for taking my suggestion!

**Comment by Jozufozu** [comment]
Thank you two!

**Pull request merged**